dev-qt/qtwebengine/qtwebengine-6.3.0.ebuild

```diff
@@ -0,0 +1,249 @@
+# Copyright 2021-2022 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+PYTHON_COMPAT=( python3_{9,10} )
+PYTHON_REQ_USE="xml(+)"
+CHROMIUM_VER="94.0.4606.126"
+CHROMIUM_PATCHES_VER="99.0.4844.84"
+
+inherit check-reqs estack flag-o-matic multiprocessing python-any-r1 qt6-build
+
+DESCRIPTION="Library for rendering dynamic web content in Qt6 C++ and QML applications"
+
+if [[ ${QT6_BUILD_TYPE} == release ]]; then
+	KEYWORDS="~amd64"
+fi
+
+IUSE="
+	alsa bindist designer geolocation +jumbo-build kerberos pipewire pulseaudio
+	+system-ffmpeg +system-icu widgets
+"
+REQUIRED_USE="designer? ( widgets )"
+
+BDEPEND="${PYTHON_DEPS}
+	$(python_gen_any_dep 'dev-python/html5lib')
```

Comment:
I think this might be missing a [${PYTHON_USEDEP}]

Reply:
Well, it was there but... see: https://github.com/gentoo/gentoo/pull/25635#discussion_r883259736

Reply:
Oh right. I'd still say it looks odd and is likely to get people asking every so often, but w/e.

Reply:
I said PYTHON_DEPS was redundant, not PYTHON_USEDEP

Reply:
Please re-read https://github.com/gentoo/gentoo/pull/25635#discussion_r883259736, and re-add USEDEP.

Reply:
Any for reference:
https://projects.gentoo.org/python/guide/any.html#dependencies
```
${PYTHON_DEPS} gets replaced by a call to python_gen_any_dep
```

Reply:
Ah, right, I don't think I read the 'BDEPEND="${PYTHON_DEPS}' line at, focusing on the line you commented on.  :]

Reply:
More specifically I selected both lines to comment on given they were related (aka any_dep provides python_deps), albeit the email version doesn't make that clear and make it look like I just picked the last line. Ah well, no biggie.

Reply:
Yeah.  It may have mostly just been me not physically really seeing the ${PYTHON_DEPS} part.  Kind of blends in, and it was a long day etc., but it definitely makes more sense now (especially the part about you not specifically mentioning USEDEP).

But yeah, will fix.